src/ziggurat/messaging/consumer.clj

```diff
@@ -25,18 +25,23 @@
   [message topic-entity]
   (try
     (s/validate mp/message-payload-schema message)
-    (catch Exception e
+    (catch Exception _
```

Comment:
Lets log the exception as well instead of ignoring it?

Reply:
done